### ISSUE TITLE
After refreshing the Sandbox, Nebula Logger logs are not generated

### ISSUE THREAD

**Issue by Mahes0990** [jongpie/NebulaLogger#482]
### Package Edition of Nebula Logger

Unlocked Package

### Package Version of Nebula Logger

4.6.15

### New Bug Summary

Hello there,

I'm Developer in my porject and recently I have noticed that when we do the sandbox refresh for other developers and they keep complaining saying that Logs are not been generated. we have did our analysis  but we could find the cause of it . before refresh it was working fine

so someone from the Nebula logger could help us!

thank you in advance

**Comment by jongpie** [comment]
@Mahes0990 can you clarify what sandbox type(s) you're using (full copy, partial copy, dev pro or dev)? And do you have records in the custom setting object `LoggerSettings__c`?

**Comment by dlfaust** [comment]
I'm just experiencing this now as well. We did a full sandbox refresh on 4/24/23 and I'm trying to generate logs today and it is not working.  It is working ok in production and in another full sandbox that has not been refreshed.  We do have a custom setting where I have changed days to retain logs to 45. 
 
![image](https://github.com/jongpie/NebulaLogger/assets/7336696/9fd8e7d5-b829-4f3d-b1de-2f66d0f38e19)
